### DISCUSSION TITLE
Surface Laptop 4, cannot control backlight

### DISCUSSION THREAD

**Discussion by iameeeom** [linux-surface/linux-surface#787] [Q&A] [answered]
i installed ubuntu 20.04,  and touchpad, keyboard work well, but cannot change the backlight brightness in nomodeset...

i tried "quiet splash acpi_backlight=vendor" instead of "quiet splash nomodeset"  to try adding acpi_backlight=vendor to the kernel parameters. but it was not boot...

i cannot work with this situation,  moniter is so bright...

how can i do that??

**Comment by tomasdziovenas** [comment, marked as answer]
Hi, after updating the grub boot file, have you done this command?
`sudo update-grub` and then restart the PC 

My /etc/default/grub file has this line, maybe you can try and use it:
`GRUB_CMDLINE_LINUX_DEFAULT="quiet splash amd_iommu=off iommu=off acpi_backlight=vendor"`

I'm using UBUNTU 21.10
